README.md

```diff
@@ -22,30 +22,61 @@ pip install mkdocs-llmstxt
 Enable the plugin in `mkdocs.yml`:
 
 ```yaml title="mkdocs.yml"
+site_name: My project
+site_description: Description of my project
+site_url: https://myproject.com/  # Required for the llmstxt plugin to work
+
 plugins:
 - llmstxt:
-    files:
-    - output: llms.txt
-      inputs:
-      - file1.md
-      - folder/file2.md
+    markdown_description: Long description of my project
+    sections:
+      Usage documentation:
+        - file1.md
+        - file2.md
+```
+
+The resulting `/llms.txt` file will be available at the root of your documentation.
+With the previous example, it will be accessible at https://myproject.com/llms.txt
+and will contain the following:
+
+```markdown
+# My project
+
+> Description of my project
+
+Long description of my project
+
+## Usage documentation
+
+- [File1 title](https://myproject.com/file1.md)
+- [File2 title](https://myproject.com/file2.md)
```

Comment:
There's currently no way to specify a description for each list element (e.g. `[File1 title](https://...): description`.

`File1 title` is inferred from the `title` property of the `Page` object, but the same cannot be done for the description.

Should we allow something like:

```markdown
plugins:
- llmstxt:
    markdown_description: Long description of my project
    sections:
      Usage documentation:
        - path: file1.md
          description: Desc
        - file2.md
```
?

Reply:
Yep, good idea, I thought of that too. I don't consider this blocking for the PR though, can be done in a subsequent one :slightly_smiling_face:

Reply:
I can open an issue about this once we merged this one :)

Reply:
That'd be great, thanks :pray: